rocprim/PKGBUILD

```diff
@@ -6,7 +6,7 @@ pkgdesc='Header-only library providing HIP parallel primitives'
 arch=('x86_64')
 url="https://github.com/ROCmSoftwarePlatform/rocPRIM"
 license=('MIT')
-depends=('hip')
+depends=('hip-hcc')
```

Comment:
Does rocprim work with hip-nvcc and hip-clang? Shouldn't this stay generic?

Reply:
rocprim only works on the ROCm platform. The generic version for ROCm and CUDA is hipcub.

Reply:
I see kk